### PR TITLE
add terrascan atlantis container files, scripts and doc.

### PR DESCRIPTION
**Pull request by dev-gaur**
#686 

To test the container :

Follow along the doc from here.
https://github.com/accurics/terrascan/pull/684/files#diff-eef1440e3de84d482add1b05dff1df7e5553757ed39a6a5ecf6007e91c4f412cR76

### CODE REVIEW

**Comment by amirbenv** [comment]
Mind having descriptive filenames? atlantis-entrypoint.sh and atlantis.sh is confusing, same with setup and setup1. Other files can also be clearer (workflow.yaml --> terrascan_workflow,yaml)

Also, maybe consolidate the 3 commands in setup.sh with the Dockerfile?

Lastly we've refactored the docs page, so instead of reuploading the old doc, edit the doc inside /docs/integrations/

**Comment by dev-gaur** [comment]
> Mind having descriptive filenames? atlantis-entrypoint.sh and atlantis.sh is confusing, same with setup and setup1. Other files can also be clearer (workflow.yaml --> terrascan_workflow,yaml)
> 
> Also, maybe consolidate the 3 commands in setup.sh with the Dockerfile?
> 
> Lastly we've refactored the docs page, so instead of reuploading the old doc, edit the doc inside /docs/integrations/

@amirbenv  Changes made

File: docs/integrations/atlantis-integration.md [hunk 1]
```diff
@@ -62,7 +61,7 @@ $ atlantis server \
 --gh-token="$TOKEN" \
 --gh-webhook-secret="$SECRET" \
 --repo-allowlist="$REPO_ALLOWLIST" \
---repo-config=terrascan-workflow.yaml
+--repo-config=terrascan-terrascan_workflow.yaml
```
Comment (amirbenv) [inline docs/integrations/atlantis-integration.md:64]: I think there's an extra "terrascan-" in there

**Review by jlk — COMMENTED** [review]
I don't see anything related to building `accurics/terrascan_atlantis` or pushing it to docker hub?

**Comment by dev-gaur** [comment]
Blocked on #705

**Comment by dev-gaur** [comment]
For verifying the output :

Incase of no violations from terrascan: https://github.com/dev-gaur/atlantis-demo/pull/3#issuecomment-831188764 

(basically no comments from terrascan). The CI passes.

Incase of 1 or more violations from terrascan: https://github.com/dev-gaur/atlantis-demo/pull/3#issuecomment-831044692 

( THE CI WILL BREAK )

To test the container yourself, you can follow along the docs I added on this PR. or try 
```
docker pull devanggaur7/terrascan_atlantis:v0.1.0 
```

File: docs/integrations/atlantis-integration.md [hunk 2]
```diff
@@ -74,6 +73,40 @@ terrascan server
...
 
-## Custom Atlantis Contaier
-(coming soon...)
+## Custom Atlantis Container
+
+### Usage
+
+To use our container image:
+```
+docker pull accurics/terrascan_atlantis:v0.1.0
```
Comment (jlk) [inline docs/integrations/atlantis-integration.md:82]: on versioning...

I'd think we version the container image the same as terrascan releases, just to be consistent?

Also here, maybe specify terrascan_atlantis:latest, and refer users to the tags tab of the docker hub page if they want to find specific versions?

**Comment by jlk** [comment]
Besides the comment on the version stuff in the docs, I think this is good to go.

File: docs/integrations/atlantis-integration.md (hunk 2 above)
Comment (dev-gaur) [reply docs/integrations/atlantis-integration.md:82]: > I'd think we version the container image the same as terrascan releases, just to be consistent?

then versions of atlantis releases should also be considered?

**Review by jlk — COMMENTED** [review]
Hmm...I as new versions of helm/terraform/kustomize/etc come out, we usually have to do a new release to support, so probably the same for Atlantis?

File: docs/integrations/atlantis-integration.md (hunk 2 above)
Comment (jlk) [reply docs/integrations/atlantis-integration.md:82]: Hmm...I as new versions of helm/terraform/kustomize/etc come out, we usually have to do a new release to support, so probably the same for Atlantis?

**Comment by dev-gaur** [comment]
@jlk I updated the doc once more. please check

**Review by jlk — APPROVED** [review]
LGTM

**Pull request merged**